hybrid/DockerfileFreeRadius

```diff
@@ -1,19 +1,41 @@
+# FreeRADIUS Dockerfile for WBA OpenRoaming Connector (Hybrid Mode)
+# Based on freeradius/freeradius-server but configured for OpenRoaming IDP
+
 FROM freeradius/freeradius-server:3.2.1
```

Comment:
We need to qualify this Docker file because the assumption is that this will be run on Intel (x86 / x64). This Docker config won't work on Raspberry Pi (ARMv8 / arm64). Unfortunately, FreeRADIUS project does not build arm64 binaries and I don't see any configs on Docker Hub that do arm64. 

Other than that, this is great work!